### PR TITLE
Update bookings-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Added steps to connect bookings to Facebook as the Bookings Page redirects users to this document when they are trying to connect to Facebook

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3029

File: microsoft-365/bookings/bookings-overview.md [hunk 1]
```diff
@@ -25,6 +25,20 @@ Bookings has three primary components:
 
 - A business-facing mobile app where Bookings calendar owners and administrators can see all of their appointments, access customer lists and contact information, and make manual bookings on the go.
 
+## Connect Microsoft Bookings to Facebook Page.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-overview.md]:
```suggestion
## Connect Microsoft Bookings to your Facebook page
```

File: microsoft-365/bookings/bookings-overview.md [hunk 2]
```diff
@@ -25,6 +25,20 @@ Bookings has three primary components:
...
+
+1)To remove or connect Microsoft Bookings and Facebook page, open your business Facebook page.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-overview.md]:
```suggestion
1. To remove or connect Microsoft Bookings and your Facebook page, open the Facebook page of your organization.
```

File: microsoft-365/bookings/bookings-overview.md [hunk 3]
```diff
@@ -25,6 +25,20 @@ Bookings has three primary components:
...
+
+2)Select **Settings** from the upper right corner and chose **Partner Apps and Services** option from the left panel.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-overview.md]:
```suggestion
2. Select **Settings** from the upper right corner and chose the **Partner Apps and Services** option from the left panel. 
```

File: microsoft-365/bookings/bookings-overview.md [hunk 4]
```diff
@@ -25,6 +25,20 @@ Bookings has three primary components:
 
 - A business-facing mobile app where Bookings calendar owners and administrators can see all of their appointments, access customer lists and contact information, and make manual bookings on the go.
 
+## Connect Microsoft Bookings to Facebook Page.
+
+1)To remove or connect Microsoft Bookings and Facebook page, open your business Facebook page.
+
+2)Select **Settings** from the upper right corner and chose **Partner Apps and Services** option from the left panel. 
+
+3)Next hit the **Add Service** button, select **Microsoft Bookings** and click **Add Service** option.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-overview.md]:
```suggestion
3. Next, select the **Add Service** button, choose **Microsoft Bookings**, and select the **Add Service** option.
```

File: microsoft-365/bookings/bookings-overview.md [hunk 5]
```diff
@@ -25,6 +25,20 @@ Bookings has three primary components:
...
+
+4)Instantly, a new tab will open. When it is visible, sign into your Office 365 account and find your booking calendars. Here, select the calendar you want to connect to your Facebook page and choose the Connect option.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-overview.md]:
```suggestion
4. A new tab will open instantly. When it is visible, sign in to your Office 365 account and find your Bookings calendars. Select the calendar you want to connect to your Facebook page and choose the **Connect** option.
```

File: microsoft-365/bookings/bookings-overview.md [hunk 6]
```diff
@@ -25,6 +25,20 @@ Bookings has three primary components:
...
+
+5)Once you complete the above step, the tab will close, and you’ll return to Facebook. Now you’re all set to receive Bookings through your Facebook page.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-overview.md]:
```suggestion
5. Once you've completed the previous step, the tab will close and you’ll be returned to Facebook. Now you’re all set to receive Bookings through your Facebook page.
```

File: microsoft-365/bookings/bookings-overview.md [hunk 7]
```diff
@@ -25,6 +25,20 @@ Bookings has three primary components:
...
+
+6)If you would like to remove Microsoft Bookings and Facebook page, simply select Remove Service on the Page Action button in the Microsoft Bookings tab.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-overview.md]:
```suggestion
6. If you would like to remove Microsoft Bookings from your Facebook page, simply select **Remove Service** on the **Page Action** button in the Microsoft Bookings tab.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by JohanFreelancer9** [comment]
@msbemba Please look at the rich diff. The numbering is not formatted correctly.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder please merge this PR. Thanks.

**Pull request closed without merging**